/swiss_exception.cpython-35.pyc
